block/test/cardano-sl-block-test.cabal

```diff
@@ -0,0 +1,83 @@
+name:                cardano-sl-block-test
+version:             1.1.0
+synopsis:            Cardano SL - block processing (tests)
+description:         QuickCheck Arbitrary instances for Cardano SL block
+                     processing.
+license:             MIT
+license-file:        LICENSE
+author:              Serokell
+maintainer:          hi@serokell.io
+copyright:           2018 IOHK
+category:            Currency
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  exposed-modules:
+                       Test.Pos.Block.Arbitrary
+                       Test.Pos.Block.Arbitrary.Message
+                       Test.Pos.Block.Arbitrary.Generate
+
+  build-depends:       QuickCheck
+                     , base
+                     , bytestring
+                     , cardano-sl-binary
+                     , cardano-sl-block
+                     , cardano-sl-core
+                     , cardano-sl-core-test
+                     , cardano-sl-crypto
+                     , cardano-sl-db
+                     , cardano-sl-delegation
+                     , cardano-sl-infra
+                     , cardano-sl-lrc
+                     , cardano-sl-lrc-test
+                     , cardano-sl-networking
+                     , cardano-sl-ssc
+                     , cardano-sl-txp
+                     , cardano-sl-txp-test
+                     , cardano-sl-update
+                     , cardano-sl-util
+                     , cardano-sl-util-test
+                     , formatting
+                     , generic-arbitrary
+                     , quickcheck-instances
+                     , random
+                     , text
+                     , text-format
+                     , universum
+
+  default-language:    Haskell2010
+
+  default-extensions:  DeriveDataTypeable
```

Comment:
As we create new packages it would be good to reduce these `default-extensions` lists, if not remove them. In this case can you remove the whole thing and add per module `{-# LANGUAGE ExtensionName #-}` pragmas to the top of the file. These should be the first thing in the file and should be separated from `{-# OPTIONS_GHC ... #-}` pragmas by an empty line.